chronicle/release/releasers/github/summarizer.go

```diff
@@ -160,25 +161,29 @@ func (s *Summarizer) Changes(sinceRef, untilRef string) ([]change.Change, error)
 		changes = append(changes, changesFromStandardPRFilters(s.config, allMergedPRs, sinceTag, untilTag, includeCommits)...)
 	}
 
+	allClosedIssues, err := fetchClosedIssues(s.userName, s.repoName)
```

Comment:
There is a possibility that this fetch call runs and the data is never used if configurations are one of the following:
* IncludeIssues = false and IncludeUnlabeledIssues = false
* IncludeIssues = true and IssuesRequireLinkedPR = true and IncludeUnlabeledIssues = false

I don't really think this is a problem, but I can optimize this with some more if statements if it's a concern.